src/command_modules/azure-cli-servicebus/azure/cli/command_modules/servicebus/commands.py

```diff
@@ -0,0 +1,132 @@
+    )
+
+    sb_queue_util = CliCommandType(
+        operations_tmpl='azure.mgmt.servicebus.operations.queues_operations#QueuesOperations.{}',
+        client_factory=queues_mgmt_client_factory
+    )
+
+    sb_topic_util = CliCommandType(
+        operations_tmpl='azure.mgmt.servicebus.operations.topics_operations#TopicsOperations.{}',
+        client_factory=topics_mgmt_client_factory
+    )
+
+    sb_subscriptions_util = CliCommandType(
+        operations_tmpl='azure.mgmt.servicebus.operations.subscriptions_operations#SubscriptionsOperations.{}',
+        client_factory=subscriptions_mgmt_client_factory
+    )
+
+    sb_rule_util = CliCommandType(
+        operations_tmpl='azure.mgmt.servicebus.operations.rules_operations#RulesOperations.{}',
+        client_factory=rules_mgmt_client_factory
+    )
+
+    sb_geodr_util = CliCommandType(
+        operations_tmpl='azure.mgmt.servicebus.operations.disaster_recovery_configs_operations#DisasterRecoveryConfigsOperations.{}',
+        client_factory=disaster_recovery_mgmt_client_factory
+    )
+
+# Namespace Region
+    custom_tmpl = 'azure.cli.command_modules.servicebus.custom#{}'
+    servicebus_custom = CliCommandType(operations_tmpl=custom_tmpl)
+    with self.command_group('servicebus namespace', sb_namespace_util, client_factory=namespaces_mgmt_client_factory) as g:
+        g.custom_command('create', 'cli_namespace_create')
+        g.command('show', 'get', exception_handler=empty_on_404)
+        g.custom_command('list', 'cli_namespace_list', exception_handler=empty_on_404)
+        g.command('delete', 'delete')
+        g.command('exists', 'check_name_availability_method')
+        g.generic_update_command('update', custom_func_name='cli_namespace_update', custom_func_type=servicebus_custom)
+
+    with self.command_group('servicebus namespace authorization-rule', sb_namespace_util, client_factory=namespaces_mgmt_client_factory) as g:
+        g.command('create', 'create_or_update_authorization_rule',)
+        g.command('show', 'get_authorization_rule', exception_handler=empty_on_404)
+        g.command('list', 'list_authorization_rules', exception_handler=empty_on_404)
+        g.command('keys list', 'list_keys')
+        g.command('keys renew', 'regenerate_keys')
+        g.command('delete', 'delete_authorization_rule')
+        g.generic_update_command('update', getter_name='get_authorization_rule', setter_name='create_or_update_authorization_rule', custom_func_name='cli_namespaceautho_update')
+
+# Queue Region
+    with self.command_group('servicebus queue', sb_queue_util, client_factory=queues_mgmt_client_factory) as g:
+        g.custom_command('create', 'cli_sbqueue_create')
+        g.command('show', 'get', exception_handler=empty_on_404)
+        g.command('list', 'list_by_namespace', exception_handler=empty_on_404)
+        g.command('delete', 'delete')
+        g.generic_update_command('update', custom_func_name='cli_sbqueue_update')
```

Comment:
Several help entries state that they have default values.  They should not. Update arguments should never have default values.

Reply:
implemented the update command separately, parameter help text doesn't have the default values. The create method parameter help text has default values.